notebooks/Beta_Visualization.ipynb

```diff
@@ -0,0 +1,729 @@
+{
```

Comment:
<p>Rename notebook to 08_practical_methods__information_geometry_beta.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>